include/os/freebsd/zfs/sys/zfs_context_os.h

```diff
@@ -45,6 +45,8 @@
 #define	HAVE_LARGE_STACKS	1
 #endif
 
+#define	THREAD_SIZE	KSTACK_PAGES * PAGE_SIZE
```

Comment:
This file should include "opt_kstack_pages.h" before other includes to make sure it captures any differently-configured stack sizes. You'll need to add `opt_kstack_pages.h` to `SRCS` in `Makefile.bsd` so that the kmod.mk infrastructure generates a new one, but it should work smoothly.

Reply:
actually, that's maybe still too late. It needs to be before we could possibly include machine/param.h